event_track_presence_claim/README.rst

```diff
@@ -0,0 +1,21 @@
+.. image:: https://img.shields.io/badge/licence-AGPL--3-blue.svg
+    :target: http://www.gnu.org/licenses/agpl-3.0-standalone.html
+    :alt: License: AGPL-3
+
+==========================
+Event track presence claim
+==========================
+
+* Planner who creates a claim, and sends an email, when the teacher does not
+  impute hours in the sessions, and it is the third warning.
+
```

Comment:
Yo en la descripción pondría algo así:

This module adds a planner to control if the teacher does not register the session hours. When the teacher has been warned three times, the system will automatically create a claim and sent an email with the reason.

Reply:
Realizada la mejora solicitada por @Daniel-CA , y subido el módulo. Eskerrik asko.